### ISSUE TITLE
✅ 📊 🐞 Bug Report: Cache issue with stable feature: "Improved CSS Loading" (Related to #23821) [ED-15215] 

### ISSUE THREAD

**Issue by franzarmas** [elementor/elementor#21879]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I have enabled the "Improved CSS Loading" feature and I've noticed that it injects <link rel="stylesheet"...> into the source code. However, it injects it without a 'query string'.

For example:

- Currently injects:
```
<link rel="stylesheet" href="https://site.com/wp-content/plugins/elementor/assets/css/widget-icon-list.min.css">
```

- Should inject:
```
<link rel="stylesheet" href="https://site.com/wp-content/plugins/elementor/assets/css/widget-icon-list.min.css?ver=3.12.0">
```
(Note that ?ver=3.12.0 was added)

This to break the cache when the css files change after an update. On sites that use CloudFlare this is a problem since css file caching is enabled by default.

### Steps to reproduce

Enabled the "Improved CSS Loading" feature.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.55 (Unix) OpenSSL/3.0.2
	MySQL version: mariadb.org binary distribution v10.6.12
	PHP Version: 7.4.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%
	Language: en-US
	Timezone: America/Lima
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

== Active Plugins ==
	Add Categories to Pages.
		Version: 1.2
		Author: a.ankit

	Advanced Custom Fields PRO
		Version: 6.0.7
		Author: WP Engine

	Disable Emojis (GDPR friendly)
		Version: 1.7.4
		Author: Ryan Hellyer

	Disable Gutenberg
		Version: 2.9
		Author: Jeff Starr

	Elementor
		Version: 3.12.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.0
		Author: Elementor.com

	Email Address Encoder
		Version: 1.0.22
		Author: Till Krüss

	Facebook Chat Plugin - Live Chat Plugin for WordPress
		Version: 2.5
		Author: Meta

	MainWP Child
		Version: 4.4.0.4
		Author: MainWP

	Purify WordPress Menus
		Version: 3.4.1
		Author: Kybernetik Services

	Remove jQuery Migrate
		Version: 1.0.3
		Author: Hendy Tan

	Ultimate Addons for Elementor
		Version: 1.36.14
		Author: Brainstorm Force


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Inactive
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Active
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible
	Ultimate Addons for Elementor: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Ultimate Addons for Elementor: Incompatible

````

</details>

**Comment by shimsa** [comment]
This is a real problem. Please fix it as soon as possible.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.23.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
